tools/distrib/python/submit.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
```

Comment:
I don't really see the point in making this a Python script instead of a shell one, but whatever floats your boat, I guess.